### DISCUSSION TITLE
V3如何部署到render平台？

### DISCUSSION THREAD

**Discussion by meowluna** [AlistGo/alist#1800] [Q&A]
v3版本的环境变量名称是否和v2版本一致？我在照抄v2的环境变量后不生效
![image](https://user-images.githubusercontent.com/62191675/192127821-596b673d-5e05-4097-8e8f-7d6853b0a252.png)

**Comment by xhofe** [comment]
https://github.com/alist-org/alist/blob/main/internal/conf/config.go
env中即为环境变量名。

**Reply by zhangshiha0** [reply]
大佬我用你更新过的那个alist-render一键部署，不用远程数据库的话可以部署成功，但在环境变量里使用远程数据库就无法部署![IMG_20220926_135339.jpg](https://user-images.githubusercontent.com/46449269/192203155-8ce328b0-d4b2-4476-8456-ad5abd0d23e9.jpg)这个是使用远程数据库部署后一直是这几句输出循环
![IMG_20220926_135358.jpg](https://user-images.githubusercontent.com/46449269/192203276-de60c0a8-5eb9-4437-85fd-1930e1b0688d.jpg)这个是环境变量

**Reply by meowluna** [reply]
![image](https://user-images.githubusercontent.com/62191675/192287549-d23ed145-b0ba-4da5-9364-5e4906e748bd.png)
![image](https://user-images.githubusercontent.com/62191675/192294601-5af671e0-e7eb-4d6e-b792-76939006163e.png)

不大行

**Reply by xing00you** [reply]
只写4个参数，其他不写，可以。

**Reply by xing00you** [reply]
另外这个profix参数是不是alist_等待大佬解释

**Reply by zhangshiha0** [reply]
> 只写4个参数，其他不写，可以。

4个？哪四个，数据库参数应该都不止四个吧。其实什么都不填是可以部署的，但会丢失数据

**Reply by meowluna** [reply]
> 只写4个参数，其他不写，可以。

难道不会丢数据吗

**Reply by xing00you** [reply]
我也不太懂，会丢数据，找个网络监控，一直盯着。

**Reply by alhricke** [reply]
同样是按照v2的变量配置，重新配置v3，使用远程数据库就会出错，去掉所有变量反而可以成功的运行起来

**Reply by zhangshiha0** [reply]
> 同样是按照v2的变量配置，重新配置v3，使用远程数据库就会出错，去掉所有变量反而可以成功的运行起来

是的

**Reply by Lemon7ProPlus** [reply]
SSL mode吧.....

**Reply by PennHui666** [reply]
webdav连不上你们能连上吗？

**Reply by wxydejoy** [reply]
> webdav连不上你们能连上吗？

不行的
